### PR TITLE
(fix) Vitals form should not render if metadata is loading

### PR DESCRIPTION
**Pull request by ibacher**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

I haven't found a way to reproduce this issue other than the e2e tests on CI, so I've created a very minimal dummy CI.

Basically this does two things:

1. Makes min / max properties of the various `NumberInput` components we use work if there are no minimum or maximum values
2. If the form tries to render while the concept metadata is being loaded, we render a small skeleton instead.

Goal here is to fix the failing e2e tests.

### CODE REVIEW

**Review by denniskigen — APPROVED** [review]
LGTM! Thanks, @ibacher
